src/contracts/core/StrategyManager.sol

```diff
@@ -175,35 +175,19 @@ contract StrategyManager is
 
     /**
      * @notice Records an overcommitment event on behalf of a staker. The staker's beaconChainETH shares are decremented by `amount`.
-     * @param overcommittedPodOwner is the pod owner to be slashed
+     * @param podOwner is the pod owner to be slashed
      * @param beaconChainETHStrategyIndex is the index of the beaconChainETHStrategy in case it must be removed,
-     * @param amount is the amount to decrement the slashedAddress's beaconChainETHStrategy shares
+     * @param currentAmount is the existing amount of beaconchain ETH shares in the strategy,
+     * @param newAmount is the new amount of beaconchain ETH shares in the strategy,
      * @dev Only callable by EigenPodManager.
      */
-    function recordOvercommittedBeaconChainETH(address overcommittedPodOwner, uint256 beaconChainETHStrategyIndex, uint256 amount)
+    function recordBeaconChainETHBalanceUpdate(address podOwner, uint256 beaconChainETHStrategyIndex, uint256 currentAmount, uint256 newAmount)
```

Comment:
`amountToRemove` and `amountToAdd` are clearer

Reply:
Hmm ok, if you look at it, I'm only using the difference of the two values to either increase or decrease shares.

Reply:
got it. that's fine then